### DISCUSSION TITLE
Using shift as a modifier

### DISCUSSION THREAD

**Discussion by MattyGWS** [gerritdevriese/kzones#20] [Ideas]
Hi, I'm quite used to fancy zones but sometimes I don't want to snap to a zone, so I have it set so that I need to hold shift while dragging a window for it to snap. I think this would be a great additional feature to kzones but I'm not skilled enough to add it myself so I figured I'd start a discussion to see what you guys think. :)

Kzones is awesome by the way! Thank you for developing it.
Matty

**Comment by gerritdevriese** [comment]
What a coincidence 😯, I have been wanting this for a while but never could figure out how to do this until... yesterday.
It's already in the test branch: https://github.com/gerritdevriese/kzones/commit/74262853a5f6791a75fbbde35e4c8ac60c8209ba

There are still some issues with Wayland (as usual) but I hope that's got something to do with the mouse and not the keystate engine.

**Comment by MattyGWS** [comment]
:o read my mind I see. I can't wait to use this in action! Thanks :)

**Comment by Farmerbrown87** [comment]
Hi, 

I'm also keen to see a shift key modifier. I'm new to using KWin scripts, so I'll wait for this feature to hit the 'main' fork. 

Loving KZones, thank you so much.
